instrumentation/grpc-1.5/testing/src/main/groovy/io/opentelemetry/instrumentation/grpc/v1_5/AbstractGrpcTest.groovy

```diff
@@ -323,25 +324,26 @@ abstract class AbstractGrpcTest extends InstrumentationSpecification {
     def port = PortUtils.randomOpenPort()
     Server server
     server = configureServer(ServerBuilder.forPort(port)
-      .addService(greeter))
+      .addService(greeter)
       .intercept(new ServerInterceptor() {
         @Override
         <ReqT, RespT> ServerCall.Listener<ReqT> interceptCall(ServerCall<ReqT, RespT> call, Metadata headers, ServerCallHandler<ReqT, RespT> next) {
-          def ctx = Context.current().withValue(key, "meow")
-          def oldCtx = ctx.attach()
```

Comment:
This was wrong in my previous change, naturally just like TracingServerInterceptor, any interceptor that modifies context must use `Contexts.interceptCall` to make sure it's propagated.

Reply:
Incidentally this is much easier when context is mutable :)